01_fib/hsy3418/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int) {
+	first := 1
+	second := 1
```

Comment:
You can do these two assignments in one line: `first, second := 1, 1`. This will then match the assignment you do further down when you re-assign them. It is also more visually pleasing as you see the first two numbers of the fibonacci sequence in a sequence.

Reply:
Done.